examples/abitest/config/config.textproto

```diff
@@ -26,6 +26,19 @@ node_configs {
     address: "test.invalid:9999"
   }
 }
+node_configs {
+  name: "grpc-client"
+  grpc_client_config {
+    address: "localhost:7878"
+  }
+}
+node_configs {
+  name: "absent-grpc-client"
+  grpc_client_config {
+    # TODO(google/asylo#NNN): use test.invalid when Asylo crash fixed.
```

Comment:
Is there an actual issue to link here? If not, should we create one?

Reply:
In progress &ndash; I'm trying to get a simple repro scenario (and there appear to be two different crashes in Asylo, one easy to fix and one not so much).